lib/src/RemoteWriter.cpp

```diff
@@ -37,17 +64,25 @@ RemoteWriter::writeToDevice()
 
     QString s;
     mutex.lock();
-    if (!m_logIsPaused && m_Socket->state() == QAbstractSocket::ConnectedState)
-    {
-        int msgcount = m_logMessageList.count();
-        for (int i=0; i<msgcount; i++) {
-            s = m_logMessageList.takeFirst().message();
-            m_Socket->write(s.toLatin1()+"\r\n");
+    if (!m_logIsPaused) {
+        if (m_Config.netProtocol == UNQL::UDP) {
+            int msgcount = m_logMessageList.count();
+            for (int i=0; i<msgcount; i++) {
+                s = this->getMessage();
+                int wb = m_pUdpSocket->writeDatagram(s.toLatin1()  +"\r\n", QHostAddress(m_serverAddress), m_serverPort);
```

Comment:
why do you use `toLatin1` instead of `toUtf8`?

Reply:
toLatin1() was the usual / suggested replacement from toAscii() that was used throughout the lib. I have nothing against changing all to UTF-8 but i would do it in one different MR